browser/templates/base.html

```diff
@@ -48,7 +50,7 @@
     <!-- Load JQuery -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <!-- Load Materialize JS -->
-    <script type="text/javascript" src="{{ url_for('static', filename='js/materialize.min.js') }}"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/dropzone/5.7.2/min/dropzone.min.js"></script>
```

Comment:
I think we need to keep materialize here and load dropzone in the `extraJs` block in the template that uses DropZone.